pkg/core/resources/model/resource.go

```diff
@@ -461,7 +468,14 @@ func ComputeLabels(r Resource, mode config_core.CpMode, isK8s bool, systemNamesp
 		}
 	}
 
-	if ns, ok := labels[mesh_proto.KubeNamespaceTag]; ok && r.Descriptor().IsPolicy && r.Descriptor().IsPluginOriginated {
```

Comment:
we should refactor computeLabals, to be more specific and probably use some form of builder